docs/reference-guide/13-file-format.md

```diff
@@ -2,14 +2,27 @@
 This is a list of all recent changes that came with new Antares Simulator features. The main goal of this document is to lower the costs of changing existing interfaces, both GUI and scripts.
 
 ## v8.4.0
-### Input
+### **Writing mps files**
+mps files of first optimization used to be overwritten by mps files of second optimization. Not anymore.
+Now user can choose to print :
+- no mps file,
+- mps files related to the first opimization,
+- mps files related to the second opimization,
+- mps files related to the both opimizations.
+
+In the **generaldata.ini** input file, corresponding values for **include-exportmps** are : **none**, **optim-1**, **optim-2**, **both-optims**.
+
+Values **none** and **both-optims** can be replaced respectively by **false** and **true**, so that Antares Simulator **v8.4.0** can read studies of older versions.
+
+### **Marginal price of a binding constraint**
+**Input** :
 In the context of the addition of a new output variable (marginal price associated to a binding constraint), file **input/bindingconstraints/bindingconstraints.ini** get 2 new parameters for each binding constraint.
 They constrol which marginal price time granularity is printed, either regarding year by year or synthesis results.
 
 * `filter-year-by-year`. Default value = hourly, daily, weekly, monthly, annual
 * `filter-synthesis`. Default value = hourly, daily, weekly, monthly, annual
 
-### Output
+**Output** :
```

Comment:
No need to make titles bold

```md

# Title 1

## Title 2

### Title 3

#### Title 4
```

# Title 1

## Title 2

### Title 3

#### Title 4

Reply:
Done